src/components/SearchResult.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component } from 'react';
+import { ISearchResult } from '../types';
+
+export default class SearchResult extends Component<ISearchResult> {
+  render() {
+    const { data } = this.props;
+    return (
+      <div className="col">
+        {data.map((item) => (
+          <div className="card" key={item.uid}>
+            <b>{item.title}</b>
+            <ul>
+              <li>
+                {item.numberOfPages
+                  ? `Number of pages: ${item.numberOfPages}`
+                  : ''}
+              </li>
+              <li>
+                {item.numberOfPages
```

Comment:
This check is quite strange. Why we don't show published date if there is no `numberOfPages`?